packages/resolver/lib/resolver.ts

```diff
@@ -16,7 +13,10 @@ export class Resolver {
   options: any;
   sources: ResolverSource[];
 
-  constructor(options: any, resolverOptions: ResolverOptions = {}) {
+  constructor(
+    options: any,
+    resolverOptions: ResolverOptions = { includeTruffleSources: true }
```

Comment:
do we really want to include Truffle sources by default? if so, I would say we should just get rid of `includeTruffleSources` (and the default here) completely and just add them

Reply:
I think this should be in a separate PR.

Reply:
I opened #5771 to track this.